src/BuildProcess/BuildContainerImage.php

```diff
@@ -55,13 +62,16 @@ class BuildContainerImage
      * Create a new project builder.
      *
      * @param  string|null  $environment
-     * @param  array  $buildArgs
+     * @param  array  $cliBuildOptions
+     * @param  array  $cliBuildArgs
+     * @param  array  $manifestBuildArgs
      * @return void
      */
-    public function __construct($environment = null, $cliBuildArgs = [], $manifestBuildArgs = [])
+    public function __construct($environment = null, $cliBuildOptions = [], $cliBuildArgs = [], $manifestBuildArgs = [])
```

Comment:
Can you move arguments in front of options in this signature, and everything else it is used? To me arguments should come before options.